### DISCUSSION TITLE
Modify counter behaviour

### DISCUSSION THREAD

**Discussion by garceger** [vectordotdev/vector#15600] [Q&A] [answered]
Is there any way to create a counter with "log_to_metric" that gets the sum of a value instead of getting the number of occurrences of a log? I have a log with turnAroundTime that I want to have in counter format

**Comment by jszwedko** [comment, marked as answer]
Hey @garceger !

Yeah, you should be able to sum the values. See this example: https://vector.dev/docs/reference/configuration/transforms/log_to_metric/#examples-sum
